templates/store/search.html

```diff
@@ -3,80 +3,89 @@
 {% block title %}Search results for {{ context.query }}{% endblock %}
 
 {% block content %}
-
-<div class="p-strip">
-  <div class="row">
-    <div class="col-12 search-results__overview">
-      Your search for <strong>&lsquo;{{ context.query }}&rsquo;</strong> returned
-      {{ context.results_count }} result{{ context.results_count|pluralize }}.
+{% if context.results_count > 0 %}
```

Comment:
The code below this if statement is all just indentation changes (up to my next comment).